app/views/admin/disputes/_form.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= form_for([:admin, @dispute], html: { class: 'form-horizontal' }) do |f| %>
+    <%= render 'shared/full_errors', object: @dispute %>
+
+    <div class="row">
+        <div class="col-md-8">
+            <div class="panel panel-default">
+                <div class="panel-heading clearfix">
+                    <div class="pull-left">
+                        <%= t(:general) %>
+                    </div>
+                </div>
+                <div class="panel-body">
```

Comment:
If next div is inside this one, maybe we could show it by indentation?
That's why I personally prefer haml or maybe slim- it's a bit easier to read, IMO.

Reply:
Both `panel-body` and` panel-heading clearfix` divs are supposed to same-level children of `panel panel-default`

Reply:
Nvm, got what you meant. Thanks!